File: beeny/PR_완주하지 못한 선수_hash.py
def solution(participant, completion):
    answer = ''
    participant.sort()
    completion.sort()
    #print(participant)
    #print(completion)
    for i in range(len(completion)):
        if completion[i] != participant[i]:
            answer = participant[i]
            break
    #print(answer)
    if answer == '':
        answer = participant[-1]
    return answer

File: beeny/PR_완주하지 못한 선수_hash.js
function solution(participant, completion) {
    
    var comp = completion.length;
    var i;
    participant.sort();
    completion.sort();
    var answer = participant[participant.length-1];
    console.log(participant);
    console.log(completion);
    
    for(i= 0; i<comp ; i++){
        if(completion[i] != participant[i]){
            answer = participant[i];
            return answer;
        }
    }
    return answer;
}